GitHubConfiguration.ps1

```diff
@@ -146,6 +146,11 @@ function Set-GitHubConfiguration
         created/updated and the specified configuration changes will only remain in memory/effect
         for the duration of this PowerShell session.
 
+    .PARAMETER GitHubBaseUrl
+        The base URL of the GitHub instance to communicate with. Defaults to 'github.com'. Provide
```

Comment:
I think the description here should be updated to explicitly indicate that users should leave off `api.` from the host name in the example that you provided.

Reply:
Clarified do-nots and a clear example.